### ISSUE TITLE
Promote a new activation function recently developed by Kuangshi technology!!!!

### ISSUE THREAD

**Issue by Lornatang** [ultralytics/yolov5#1639]
## 🚀 Feature

ReLU and PReLU are extended to 2D activation functions by adding negligible space condition overhead.

## Motivation

Can a visual task specific activation function be designed?

## Pitch

I would like to suggest a branch, but because the work is too busy, directly paste the code. It can be used directly.

## Alternatives

None.

## Additional context

```python3
import torch
import torch.nn as nn
import torch.nn.functional as F

class FReLU(nn.Module):
    r""" Applies the FReLU function element-wise.

    `"Funnel Activation for Visual Recognition" <https://arxiv.org/pdf/2007.11824.pdf>`_

    Examples:
        >>> channels = 64
        >>> frelu = FReLU(channels)
        >>> input = torch.randn(1, channels, 64, 64)
        >>> output = frelu(input)
    """

    def __init__(self, channels):
        super().__init__()
        self.FReLU = nn.Sequential(
            nn.Conv2d(channels, channels, kernel_size=3, stride=1, padding=1, groups=channels, bias=False),
            nn.BatchNorm2d(channels)
        )

    def forward(self, input: Tensor):
        out = self.FReLU(input)
        return torch.max(input, out)
```
Thank you very much for your long-term promotion of Yolo technology. I will submit some code after a while. Good luck to you!

**Comment by glenn-jocher** [comment]
@Lornatang hey thanks for the suggestion! Yes, FReLU is an interesting idea. We did add this activation function to utils/actvations.py a few months ago. I did some experiments, and found that it tends to help the smaller models most, though the tradeoff between mAP improvements and speed/FLOPS added was unclear, as this activation function in particular adds extra computations and tends to behave a bit as it's own convolution layer. 

Perhaps a better experiment would be to replace some Conv2d() layers with this function rather than replacing existing activations (such as the current hardswish).

https://github.com/ultralytics/yolov5/blob/68e6ab668b30a6014215b94e399151f8c76e471a/utils/activations.py#L63-L72

UPDATE: Interestingly, it appears there is a bug in our implementation, as we have not set bias=False. Perhaps you could submit a PR with the correction and your code comments as you have above, as this explains much better than the current code comments.

**Comment by glenn-jocher** [comment]
TODO: remove bias from FReLU convolution.

**Comment by Lornatang** [comment]
I fixed it.

**Comment by glenn-jocher** [comment]
@Lornatang ah, no, I meant that there is a bug in our official YOLOv5 code. nn.Conv2d() defaults to bias=True, which is not required, and produces unnecessary extra operations when followed by a nn.BatchNorm2d() layer.

So your original implementation was correct to use bias=False, it is the official code that needs a PR to fix.

**Comment by Lornatang** [comment]
@glenn-jocher Ah, I was careless. I didn't flash..

**Comment by glenn-jocher** [comment]
@Lornatang this bug is fixed now in PR #1666.
